app/views/giraffe/contributeReact.scala.html

```diff
@@ -0,0 +1,69 @@
+@import com.gu.i18n.CountryGroup
```

Comment:
is there any reason we have this file back? this should now be in contribute.scala.html